src/power_divergence.jl

```diff
@@ -284,7 +287,7 @@ Implements: [`pvalue`](@ref), [`confint(::PowerDivergenceTest)`](@ref)
 
   * Agresti, Alan. Categorical Data Analysis, 3rd Edition. Wiley, 2013.
 """
-function PowerDivergenceTest(x::AbstractMatrix{T}; lambda::U=1.0, theta0::Vector{U} = ones(length(x))/length(x)) where {T<:Integer,U<:AbstractFloat}
+function PowerDivergenceTest(x::AbstractMatrix{T}; lambda::U=1.0, theta0::Vector{U} = ones(length(x))/length(x), ddof::Int64=0) where {T<:Integer,U<:AbstractFloat}
```

Comment:
```suggestion
function PowerDivergenceTest(x::AbstractMatrix{T}; lambda::U=1.0, theta0::Vector{U} = ones(length(x))/length(x), ddof::Integer=0) where {T<:Integer,U<:AbstractFloat}
```
Same elsewhere.

Reply:
Great suggestion. I wonder is we should also allow `ddof` to be a Float?

Reply:
Probably not at this stage, unless we can find a use case for it.

Reply:
I changed `doff` from `Int64` to `Integer` in all function definitions.